examples/kalshi-multi-market-feed/src/main.rs

```diff
@@ -0,0 +1,17 @@
+use execution_phase::execution_phase;
+use seda_sdk_rs::oracle_program;
+use tally_phase::tally_phase;
+
+mod execution_phase;
+mod tally_phase;
+
+#[oracle_program]
+impl PriceFeed {
```

Comment:
I personally like to match the name here to OP `impl KalshiMultiMarketFeed` but doesn't actually impact anything.